ament_java_resources/jni_library_path.dsv.template

```diff
@@ -0,0 +1 @@
+prepend-non-duplicate;@JNI_LIB_ENV_VAR@;lib/jni
```

Comment:
Just one question, `prepend-non-duplicate` is supposed work for both Linux and Windows, right?

Reply:
Yes, I'm pretty sure it's supposed to work for both.